scripts/us_census/acs5yr/subject_tables/common/base.py

```diff
@@ -0,0 +1,87 @@
+"""Base data loader for ACS subject tables"""
+
+import os
+import csv
+import json
+import pandas as pd
+
+from io import BytesIO
+from requests import get
+from zipfile import ZipFile
+
+import generateColMap
+
+
+_TEMPLATE_TMCF = """Node: E:SubjectTable->E{index}
```

Comment:
Is this just be a fixed file common for everything that we can just hand-write once? (without needing to have the script generate it)

Reply:
Correct, that is the plan to use a single tmcf is used for all variables.
In hindsight, this can be put out as a file and removed from the script, since the tmcf file is made only once.